pkg/sql/catalog/descpb/structured.go

```diff
@@ -145,6 +145,28 @@ func (c ColumnIDs) Equals(input ColumnIDs) bool {
 	return true
 }
 
+// ElementWiseEquals returns true if the input list's elements are the same as this list's elements
+func (c ColumnIDs) ElementWiseEquals(input ColumnIDs) bool {
+	if len(input) != len(c) {
+		return false
+	}
+
+	columnsLookup := map[ColumnID]bool{}
```

Comment:
let's use `util.MakeFastIntSet` here. it is ok to cast the ColumnID to int.

Reply:
Have swapped to using this.
I think I have used it effectively but please let me know if there is a better way to make use of the Set.